submissions/5mountains/dom-api-task/js/main.js

```diff
@@ -0,0 +1,104 @@
+const styles = {
+    menuLink: 'menu__link',
+    menuItem: 'menu__item',
+    heroItem: 'hero__item',
+    heroImg: 'hero__image',
+    itemIcon: 'item__icon',
+    itemTitle: 'item__caption',
+    itemPrice: 'item__price',
+    itemInfo: 'item__description'
+};
+
+async function fetchAsync () {
+    let response = await fetch('./data/data.json');
+    let data = await response.json();
+    return data;
+}   
+  
+fetchAsync()
+    .then(data => {
+        let details = createHtml(data);
+        drawPage(details);
+        toggleTabs();
+        animation();
+    })
+    .catch(reason => console.log(reason.message));
+
+function createHtml(data) {
+    const menuItems = document.createDocumentFragment();
+    const heroItems = document.createDocumentFragment();
+    data.pancakes.forEach((elem, index) => {
+        const element = elem;
+        const token = index;
```

Comment:
Why are you need this? O_o

Reply:
While writing the code, the functions did not want to work without additional variables! But, now they do))) mysticism... so, I'll remove it

Reply:
Please resolve this one when you remove them

Reply:
done (: